### ISSUE TITLE
.waves-effect bug when inside columns

### ISSUE THREAD

**Issue by pajod** [Dogfalo/materialize#3528]
In Opera, *.btn* content is simply not displayed if the *.waves-effect* class is added - however: This problem does not occur in the first column. In Firefox it works as intended.

[Screenshot](https://i.imgur.com/7pZEmup.png)
[CodePen](http://codepen.io/anon/pen/zBbBWA)

I could not find a fault with the materialize css and would assume a browser bug, but I was so far unable to create a minimal example that doesnt involve materialize.

Any suggestions welcome on how to determine the root cause, work around or gather material to file a bug with affected browsers.

**Comment by Sepehr13** [comment]
I can't see any wave effect in my browser

**Comment by pajod** [comment]
@Sepehr13 correct, i removed the javascript from the CodePen to prove that the bug is not cause by the script. You could copy them from the [template codepen](http://codepen.io/Dogfalo/pen/xbzPQV) from the CONTRIBUTING.md if you want to see effect.

Whats happening is that the waves would overflow the buttons, and thus every object having the waves-effect is set not to overflow via CSS - and the surprisingly-blank red buttons above are a side-effect of the overflow settings, not the waves effect itself.

**Comment by mvanorder** [comment]
I've just tested this in Opera 39 and got the same result, however if you change the CSS url to 0.98.0 or newer the issue goes away.  This was fixed in commit [0fdebda - Removed will change on waves due to visual artifacting on certain bro… ](https://github.com/Dogfalo/materialize/commit/53eaec41f9f29c9775d1fbf290c2f5a060fdebda)
